http_service/bugbug_http/boot.py

```diff
@@ -87,9 +90,23 @@ def extract_commit_experiences():
             )
             assert ALLOW_MISSING_MODELS
 
+    def retrieve_schedulable_tasks():
+        # Store in a file the list of tasks in the latest autoland push.
+        r = requests.get(
+            "https://hg.mozilla.org/integration/autoland/json-pushes?version=2"
+        )
+        r.raise_for_status()
+        data = r.json()
+        last_push = data["pushes"][str(data["lastpushid"])]
+        last_push_revs = last_push["changesets"][::-1]
+        with open("known_tasks", "w") as f:
+            f.write("\n".join(mozci.push.Push(last_push_revs).target_task_labels))
```

Comment:
This is fine, so treat this comment as optional..

But it's just as easy to grab this from the index. I.e:
https://firefox-ci-tc.services.mozilla.com/api/index/v1/task/gecko.v2.autoland.latest.taskgraph.decision/artifacts/public/target-tasks.json

That way you don't need to query hgmo to find the latest push. Maybe `mozci.Push` should accept an index path to a decision task and then use the `parameters.yml` to instantiate itself.

Reply:
Right, it doesn't change much as it's just done once per day basically, but it's less code, so I've done it in cbfe61c.